src/main/kotlin/io/github/gradlenexus/publishplugin/NexusRepository.kt

```diff
@@ -33,6 +36,9 @@ open class NexusRepository @Inject constructor(@Input val name: String, project:
     @Input
     val snapshotRepositoryUrl = project.objects.property<URI>()
 
+    @Input
+    val publicationType: Property<NexusPublishExtension.PublicationType> = project.objects.property<NexusPublishExtension.PublicationType>().value(NexusPublishExtension.PublicationType.MAVEN)
```

Comment:
I guess we cannot use `convention` because we need to support older versions of Gradle, right?

Reply:
exactly, convention was introudced in 5.1